site/troubleshooting.md

```diff
@@ -20,7 +20,7 @@ Check the weave container logs with
 
 A reasonable amount of information, and all errors, get logged there.
 
-The log verbosity can be increased by supplying the `-debug` flag when
+The log verbosity can be increased by supplying the flag `-log-level=debug` when
```

Comment:
`--log-level=debug`, surely. So that we don't have to change this *again* when moving to unix-getopt style option parsing.